distribution/lib/Standard/Database/0.0.0-dev/src/Data/Column_Constraint.enso

```diff
@@ -0,0 +1,26 @@
+from Standard.Base import all
+
+## ADVANCED
+   Specifies additional properties for a column created using
+   `Connection.create_table`.
+
+   The support for the constraints may vary between databases.
+type Column_Constraint
+    ## ADVANCED
+       Specifies a default value for the column.
+
+       Argument:
+       - `sql_expression`: The SQL expression to use as the default value. Note that this is a raw SQL expression, so if you want to set a default to a string, you must include the quotes. The quoting style may depend on the database. Never pass unsanitized data to this parameter.
+
+       ! SQL Injection
+
+         Since `sql_expression` is a raw SQL expression, note that malicious
+         data can cause execution of arbitrary SQL queries. Only use this
+         parameter with trusted data.
+    Default (sql_expression : Text)
```

Comment:
First I wanted to put a `value : Any` here and rely on `?` JDBC interpolations. But apparently these interpolations do not work in DDL statements... So instead I decided to go for a raw expression - this will give more flexibility as one can use features like timestamp and I hope this will make it clearer this is an advanced feature.